doc/source/reference/code_support.rst

```diff
@@ -12,66 +12,78 @@ as inputs.  Below is a list showing what level of support is provided for
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| MOAB                  |     Y      |    N/A    |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Nyx                   |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| openPMD               |     Y      |     Y     |      N     |   Y   |    Y     |    Y     |     N      | Partial     |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Orion                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| OWLS/EAGLE            |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Piernik               |     Y      |    N/A    |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Pluto                 |     Y      |     N     |      Y     |   Y   |    Y     |    Y     |     Y      | Partial     |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| RAMSES                |     Y      |     Y     |      Y     |   Y   | Y [#f2]_ |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Tipsy                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| WarpX                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
 
 .. [#f1] one-dimensional base-state not read in currently.
 .. [#f2] These handle mesh fields using an in-memory octree that has not been parallelized.
          Datasets larger than approximately 1024^3 will not scale well.
 .. [#f3] Newer versions of Castro that use BoxLib's standard particle format are supported.
-	 The older ASCII format is not.
+	     The older ASCII format is not.
```

Comment:
I think it has to match the indentation level of the text above. Tabs should be avoided
```suggestion
          The older ASCII format is not.
```